packages/suite/src/components/suite/NavigationBar/components/MainNavigation/index.tsx

```diff
@@ -158,6 +155,7 @@ const MainNavigation = (props: Props) => {
                             <ItemTitle isActive={isActive} isDisabled={isDisabled}>
                                 <Translation id={translationId} />
                             </ItemTitle>
+                            {/* if the button is disabled, diplay "SOON" badge */}
```

Comment:
```suggestion
                            {/* if the button is disabled, display "SOON" badge */}
```

Reply:
I highly suggest getting a spell checker for VSCode (most other editors have them available too). Just recently started using one and it helped me more than once to catch typos in comments, strings and even variables